src/SourceGenerators/Uno.UI.Tasks/LinkerHintsGenerator/LinkerHintGeneratorTask.cs

```diff
@@ -306,13 +306,16 @@ private void BuildReferences()
 
 			string RewriteReferencePath(string referencePath, string unoUIPackageBasePath, string unoRuntimeIdentifier)
 			{
+				const string SkiaRuntimeIdentifier = "Skia";
+				const string WebAssemblyRuntimeIdentifier = "WebAssembly";
+
 				var separator = Path.DirectorySeparatorChar;
 
-				return
-					(unoRuntimeIdentifier == "Skia" || unoRuntimeIdentifier == "WebAssembly") &&
-						referencePath.StartsWith(unoUIPackageBasePath) ?
-							referencePath.Replace($"lib{separator}netstandard2.0", $"uno-runtime{separator}{unoRuntimeIdentifier}") :
-								referencePath;
+				return (unoRuntimeIdentifier.Equals(SkiaRuntimeIdentifier, StringComparison.OrdinalIgnoreCase)
+						|| unoRuntimeIdentifier.Equals(WebAssemblyRuntimeIdentifier, StringComparison.OrdinalIgnoreCase))
+					&& referencePath.StartsWith(unoUIPackageBasePath, StringComparison.OrdinalIgnoreCase)
```

Comment:
you should move the results of the 3 conditions into intermediary variables
it is a pain to cipher `(a || b) && c` with this indentation